_layouts/home.html

```diff
@@ -1,78 +1,93 @@
 ---
 layout: default
 ---
+<div id="home">
+{% include header.html %}
 
+<div class="header-image">
```

Comment:
After thinking about, probably want to make the `<div class="header-image">` container it's own include file in Jekyll. That way it could be added to other pages easily.

`_include/header_image.html`